Formula/tarantool.rb

```diff
@@ -1,8 +1,8 @@
 class Tarantool < Formula
   desc "In-memory database and Lua application server"
   homepage "https://tarantool.org/"
-  url "https://download.tarantool.org/tarantool/2.5/src/tarantool-2.5.2.0.tar.gz"
-  sha256 "f64b8ef772d42017bd938770912b4a1727ce6302087efcd5c7faac9e425d9ac4"
+  url "https://hb.bizmrg.com/tarantool_repo/sources/2.6/tarantool-2.6.1.164.tar.gz"
+  sha256 "cf202d1a7ca2bb8cfa8bdabf2ba582aeb166ff9176a93ff542ffe8135fb66015"
   license "BSD-2-Clause"
   head "https://github.com/tarantool/tarantool.git", branch: "2.5", shallow: false
```

Comment:
Does this need to be updated to `branch: "2.6"`?

Reply:
We are near to the release, and the real tarball will be available in a day or two. To be ready for brew formula we need to check it right before final releasing. That is why I didn't check the select boxes in head conversation of PR.

Reply:
BTW, should not the branch be just `master`?

Reply:
If it's just `master` then everything after and including the first `,` in the `head` line can be dropped.

As for why it's not master, I don't know. If that's your development branch, then I guess it should be.

Reply:
Non-shallow clone is necessary, but, yep, `branch: master` can be missed.

We have a set of release branches: `1.10`, `2.5`, `2.6` (we backport bugfixes to them). Aside of this, we have `master`, where development is going on.

Given this development flow, what should be in the `head` section?

Reply:
```suggestion
  head "https://github.com/tarantool/tarantool.git", shallow: false
```
I believe this should be sufficient. Note, however, that CI does not test the `head` build. Please test it on your own if you want to make sure that it's working.